src/Framework/TestCase.php

```diff
@@ -1999,7 +1999,6 @@ private function verifyMockObjects(): void
     private function handleDependencies(): bool
     {
         if (!empty($this->dependencies) && !$this->inIsolation) {
-            $className  = \get_class($this);
```

Comment:
Is it ok to inline this variable, @sebastianbergmann?

Reply:
Why not?

Reply:
👍